modxcms/revolution
# Fix resource undefined variables notices

### What does it do?
Fixed a notices about undefined variables in the resource form.

### Why is it needed?
Describe the issue you are solving.

### How to test
1. Set log level to 3.
2. Open any resource/symlink/weblink or create a new ones.
3. Open MODX error log.
4. It should be clear.

### Related issue(s)/PR(s)
#15455
